src/structs/vpx_bool_reader.rs

```diff
@@ -222,7 +224,7 @@ impl<R: Read> VPXBoolReader<R> {
                 break;
             }
 
-            *tmp_value |= (v[0] as u64) << shift;
+            *tmp_value |= (v[0] as u32) << shift;
```

Comment:
do we now fill more frequently that before (32 bits vs 64 bits)?

Reply:
It is still "one-byte-at-a-time" like before, so I don't think it will be a problem. Effectively by coming to 32 bits I avoided 64-bit multiplication, that can be slow, but I have not tested it. I'll check.

Reply:
the only concern might be the branch predictor on the fill reader, but it should still be rare enough